podcast.xsd

```diff
@@ -176,4 +225,111 @@
     </xsd:complexType>
 </xsd:element>
 
+<xsd:element name="social">
+	<xsd:annotation>
+		<xsd:documentation>
+			Here you can define some social network of your podcast.
+		</xsd:documentation>
+	</xsd:annotation>
+	<xsd:complexType>
+		<xsd:attribute name="platform" type="Platform" use="required" />
+		<xsd:attribute name="handle" type="Name" use="optional" />
+		<xsd:attribute name="href" type="LinkHref" use="required" />
+	</xsd:complexType>
+</xsd:element>
+
+<xsd:element name="donate">
+	<xsd:annotation>
+		<xsd:documentation>
+			This tag is used to specify a link for your donation page (like Patreon).
+		</xsd:documentation>
+	</xsd:annotation>
+	<xsd:complexType>
+		<xsd:attribute name="platform" type="Platform" use="required" />
+		<xsd:attribute name="handle" type="Name" use="optional" />
+		<xsd:attribute name="href" type="LinkHref" use="required" />
+	</xsd:complexType>
+</xsd:element>
+
+<xsd:element name="shop">
+	<xsd:annotation>
+		<xsd:documentation>
+			Used to specify a link to your podcast's shop.
+		</xsd:documentation>
+	</xsd:annotation>
+	<xsd:complexType>
+		<xsd:attribute name="platform" type="Platform" use="required" />
+		<xsd:attribute name="handle" type="Name" use="optional" />
+		<xsd:attribute name="href" type="LinkHref" use="required" />
+	</xsd:complexType>	
+</xsd:element>
+
+<xsd:element name="link">
+	<xsd:annotation>
+		<xsd:documentation>
+			Used to add any link you want related to your podcast.
+		</xsd:documentation>
+	</xsd:annotation>
+	<xsd:complexType>
+		<xsd:simpleContent>
+			<xsd:extension base="LinkHref">
+				<xsd:attribute name="href" type="LinkHref" use="required"/>	
+			</xsd:extension>
+		</xsd:simpleContent>
+	</xsd:complexType>
+</xsd:element>
+
+<xsd:element name="person" type="Person" />
```

Comment:
Okay that was that. I think we have to create a `PlatformLink` type with :
```
	<xsd:complexType>
		<xsd:attribute name="platform" type="Platform" use="required" />
		<xsd:attribute name="handle" type="Name" use="optional" />
		<xsd:attribute name="href" type="LinkHref" use="required" />
	</xsd:complexType>
```

and use : `<xsd:element name="social" type="Social" />` in Person and here.

Reply:
Yes!